integration_tests/integration/cookies.spec.ts

```diff
@@ -10,28 +9,9 @@ context('Cookies', () => {
     cy.task('stubAuthToken')
   })
 
-  describe('Request Link Page', () => {
-    it('should remove cookie banner once accepted and acknowledged', () => {
-      cy.visit('/link/request-link')
-      let requestLinkPage: RequestLinkPage = Page.verifyOnPage(RequestLinkPage).clickCookieAction(
-        RequestLinkPage,
-        'reject'
-      )
-      requestLinkPage
-        .hasCookieBannerContaining('rejected')
-        .doesntHaveCookieAction('accept')
-        .doesntHaveCookieAction('reject')
-
-      requestLinkPage = requestLinkPage.clickCookieAction(RequestLinkPage, 'hide')
-
-      requestLinkPage.doesntHaveCookieBanner()
-    })
-  })
-
   describe('Cookies Policy Page', () => {
     it('should remove cookie banner once accepted and acknowledged on the cookie policy page', () => {
-      cy.visit('/link/request-link')
-      const cookiesPolicyPage: CookiesPolicyPage = Page.verifyOnPage(RequestLinkPage).clickCookieAction(
+      const cookiesPolicyPage: CookiesPolicyPage = FindRecipientByPrisonNumberPage.goToPage().clickCookieAction(
```

Comment:
All of the changes in the file are basically the same thing, in that we start the test from a different page. Previously we started on the Request Link page (which is only present if magic link auth is enabled - OTC auth does not have this page)
I've changed it to use the `FindRecipientByPrisonNumberPage.goToPage()` function which goes to the Find Recipient page after signing in via the appropriate screens (as determined by the feature flag).
After all this we then click a cookie action which is on every page, and end up on the Cookies Policy Page (which is what these tests are all about)